### PR TITLE
Create Japanese-Communist-Party.xml

### PR DESCRIPTION
**Pull request by ipaucek4680**
##### Invalid certificate
```
jcp.or.jp
hd.jcp.or.jp
```
##### 403 Forbidden (even on HTTP)
```
admin.jcp.or.jp
ftp.jcp.or.jp [1]
mail.jcp.or.jp
pop.jcp.or.jp [1]
pop3.jcp.or.jp [1]
ptv.jcp.or.jp
www.ptv.jcp.or.jp
admin.ptv.jcp.or.jp
ftp.ptv.jcp.or.jp [1]
mail.ptv.jcp.or.jp
pop.ptv.jcp.or.jp [1]
pop3.ptv.jcp.or.jp [1]
smtp.ptv.jcp.or.jp [1]
smtp.jcp.or.jp [1]

[1] Apparently these domains are not used for web hosting - they are likely to be used for Email (POP/SMTP) or FTP.
```
##### "File not found" error, unable to find any valid URL to test
`live.jcp.or.jp`

### CODE REVIEW

**Comment by pipboy96** [comment]
Also try:

```
admin.jcp.or.jp
ftp.jcp.or.jp
hd.jcp.or.jp
live.jcp.or.jp
mail.jcp.or.jp
pop.jcp.or.jp
pop3.jcp.or.jp
ptv.jcp.or.jp
www.ptv.jcp.or.jp
admin.ptv.jcp.or.jp
ftp.ptv.jcp.or.jp
mail.ptv.jcp.or.jp
pop.ptv.jcp.or.jp
pop3.ptv.jcp.or.jp
smtp.ptv.jcp.or.jp
smtp.jcp.or.jp
```

**Comment by ipaucek4680** [comment]
I have updated the pull request, please have a look.

File: src/chrome/content/rules/Japanese-Communist-Party.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--
+	Mismatched:
+		- ^ (jcp.or.jp)
```
Comment (J0WI) [inline src/chrome/content/rules/Japanese-Communist-Party.xml]: Can be redirected to `www`

File: src/chrome/content/rules/Japanese-Communist-Party.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+		- hd
```
Comment (J0WI) [inline src/chrome/content/rules/Japanese-Communist-Party.xml]:
```
mail.jcp.or.jp
ptv.jcp.or.jp
www.ptv.jcp.or.jp

**Comment by ipaucek4680** [comment]
@ivysrono 
I have updated the pull request, please have a look.

**Comment by ivysrono** [comment]
@ipaucek4680 Would you add https://github.com/EFForg/https-everywhere/pull/18291#issue-304742670 and https://github.com/EFForg/https-everywhere/pull/18291#issuecomment-518800770 to rule comments?

**Pull request merged**